### ISSUE TITLE
Map reload issue.

### ISSUE THREAD

**Issue by IMMZ** [bjorn/tiled#1001]
Steps to reproduce:
1. Open two maps
2. Choose first map (select first tab)
3. Press 'Ctrl+R' (map reload)
4. Now the second map is drawn in the first tab.

When you switch to 2nd tab and then back to 1st one - there is drawn 1st map as it should be.
Xubuntu 14.04, x86, opengl is disabled. verified with last repository version.

**Comment by bjorn** [comment]
I've never been able to reproduce this. Are you consistently able to reproduce it with the steps given, or might there be something else involved?

**Comment by IMMZ** [comment]
Yeah, it still could be noticed with Tiled '2015-09-27' version. Even don't know how I can localize that problem. I tried to enable 'opengl' but nothing changed.
